BackofficeBundle/Resources/public/coffee/setUpCallAjax.coffee

```diff
@@ -8,12 +8,14 @@ $(document).ready ->
         redirectUrl = appRouter.generateUrl 'showDashboard'
         displayMenu(redirectUrl)
       else
-        xhrFifo.push(xhr)
+        xhrFifo.push({xhr:xhr, method: settings.type.toUpperCase()})
```

Comment:
I would not put the `toUpperCase` here. You should only use it for the comparaison